test/extended/router/external_certificate.go

```diff
@@ -883,11 +884,15 @@ func patchRouteToRemoveExternalCertificate(oc *exutil.CLI, routeName string) err
 	return err
 }
 
-// isMetalJob checks if the current job is running on a metal platform.
-func isMetalJob(oc *exutil.CLI) (bool, error) {
+// isOnPremJob checks if the current job is running on an on-prem platform.
+func isOnPremJob(oc *exutil.CLI) (bool, error) {
+	onPremPlatforms := sets.NewString("metal", "openstack", "kubevirt")
```

Comment:
This may feel better defined globally for the whole file. But no need to re-commit just for this change.

Reply:
It would also make sense to move `isOnPremJob` someplace where other tests could use it, such as https://github.com/openshift/origin/blob/94a4d2a6f202f64f89a1b747b44484c358f8299d/test/extended/util/framework.go, and name it `IsOnPremPlatform`.  However, as Andrey said, no need to update the PR (and then find someone who can approve changes in `extended/util/`) if the PR is fine otherwise.